batch/manage-pool/manage-pool.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Authenticate Batch account CLI session.
+az batch account login -g myresource group -n mybatchaccount
+
+# Create a new Windows PaaS pool with 3 Standard A1 VMs.
+az batch pool create \
+    --id mypool-windows \
+    --os-family 4 \
+    --target-dedicated 3 \
+    --vm-size small \
+
+# We can add some metadata to the pool.
+az batch pool set --pool-id mypool-windows --metadata IsWindows=true VMSize=StandardA1
+
+# Retrieve a list of available IaaS images and node agent SKUs
```

Comment:
Yes I think I might break it into two scripts similar to the account create samples